src/ActionRow/index.tsx

```diff
@@ -2,17 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
+export interface ActionRowProps {
```

Comment:
[clarification] Can you expand on why the `ActionRowProps` interface is defined in both this `index.tsx` file and the `index.d.ts` file?

Reply:
Removed `.d.ts` for `ActionRow` since it has `.tsx`

Reply:
@monteri [clarification] So, the introduction of the `.d.ts` files are only necessary if the component is not already written in `.tsx`? I imagine we would eventually intend to migrate our other components' `.jsx` files to `.tsx` files. If we do that, what would happen to the `.d.ts` files? Would they remain, or get moved into the migrated `.tsx` files?

I guess I'm just wondering about consistency around where we define types across components (i.e., always included in the `.d.ts`, always included in the `.tsx`, or similar).